build/run-e2e-tests.sh

```diff
@@ -1,5 +1,127 @@
 #!/bin/bash
 
-echo "E2E TESTS GO HERE!"
+#
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and# limitations under the License.
+
+
+###!!!!!!!! On travis this script is run on the .git level
+set -e
+echo -e "E2E TESTS GO HERE!"
+
+# need to find a way to use the Makefile to set these
+REGISTRY=quay.io/open-cluster-management
+IMG=$(cat COMPONENT_NAME 2> /dev/null)
+IMAGE_NAME=${REGISTRY}/${IMG}
+BUILD_IMAGE=${IMAGE_NAME}:latest
+OPERATOR_NAME=multicluster-operators-subscription-release
+
+if [ "$TRAVIS_BUILD" != 1 ]; then
+    echo -e "Build is on Travis" 
+
+
+    echo -e "\nGet kubectl binary\n"
+    # Download and install kubectl
+    curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
```

Comment:
We probably don't want to `sudo` here for security reasons.
For something easy, can we just check if `kubectl` is available and if its not just exit with an error code.

If you really want to auto download and install. Do something like this might be better for security:
```
which kubectl 
check if $? is 0 then
KUBECTL_EXEC (or whatever name you want) = `which kubectl`
if not 0 then
mkdir -p bin 
curl
mv kubectl to bin
chmod +x bin/kubectl
KUBECTL_EXEC=realpath ./bin/kubectl
```